### ISSUE TITLE
Pro Plus Support?

### ISSUE THREAD

**Issue by TaZZaT** [spbrogan/mopeka_pro_check#5]
Happy 2023.  

I purchased a Mopeka Pro "Plus" for genmon (part of my home automation) and I can't get it to work in Genmon.  On deeper investigation, I don't believe the mopeka_pro_check library is recognizing the Mopeka Pro Plus model.  Anyway, I'm wondering if you might know if it's even possible to make this work. 

Thanks in advance...

**Comment by spbrogan** [comment]
I think it just needs to be added here: https://github.com/spbrogan/mopeka_pro_check/blob/main/mopeka_pro_check/advertisement.py#L30 and https://github.com/spbrogan/mopeka_pro_check/blob/main/mopeka_pro_check/advertisement.py#L133

This was added to the esphome component and i have been told it works fine.  You can see that the sensor id is: 0x08. 
https://github.com/esphome/esphome/blob/dev/esphome/components/mopeka_pro_check/mopeka_pro_check.h#L18

There is also another sensor from Mopeka rebranded as lippert and that is getting added to esphome too.  See PR here: https://github.com/esphome/esphome/pull/4118/files


**But this HA integration no longer works**.  Over the past year there have been major changes in ble support in Home assistant and this integration hasn't been updated.  So, if you are using a version of HA from April 2022 or newer i would expect problems.  
I plan to create a new integration that can leverage the new support but i haven't had time.  If you want to get involved or read more, I started writing a few notes down in this discussion.  https://github.com/spbrogan/sensor.mopeka_pro_check/discussions/22 


For now, the best experience is by using esphome.  I authored that component and it continues to work well for me and a number of others that I have heard from.  Here is more info about esphome support which is what i recommend at the moment. https://esphome.io/components/sensor/mopeka_pro_check.html

**Comment by TaZZaT** [comment]
Thanks for all the info/help.  I was on the right track, but missed L133 - after I posted this issue I noticed the work you had done with esphome and found the manufacturing code for the "plus" in your code.  I will give the modification a try and have a spare esp32 board and will also give that a try...  Sadly, it's going to be a couple of weeks, as I'm not at the place where the mopeka is installed.  I will also review the new support you are working on... I'm a bit of a hack so we will see if I can provide any value.  Thanks again and have an epic day.
